arangod/Cluster/DBServerAgencySync.cpp

```diff
@@ -94,10 +95,19 @@ Result DBServerAgencySync::getLocalCollections(
       return Result(TRI_ERROR_INTERNAL, "Failed to emplace new entry in local collection cache");
     }
 
+    {
+      auto [it, created] = replLogs.try_emplace(dbname, vocbase.getReplicatedLogs());
+      if (!created) {
+        LOG_TOPIC("5d5c9", ERR, Logger::MAINTENANCE)
```

Comment:
I think we should use log level WARN here, as there is nothing a user can do about this, and the program will continue to run.
```suggestion
        LOG_TOPIC("5d5c9", WARN, Logger::MAINTENANCE)
```